types/mongoose-paginate-v2/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove all these rules from file and just extends the `dtslint/dt.json`. In new definitions we need all tslint rules to be satisfied.

Reply:
I have removed all rules, except 'ban-types' because changing `Object` to `object` caused conflicts with other declarations (I think from mongodb).

Note that I'm not really well versed in typescript yet, so I hope these changes are ok.

Reply:
You can disable errors at that location instead of disabling them here.